mender/client/__init__.py

```diff
@@ -52,6 +52,18 @@ def deployments_url(host, path=''):
         return add_url_path(ap, path)
     return ap
 
+def deployments_update_url(host, id, path=''):
+    ap = add_url_path(host, service_path('/deployments/device/deployments/%s/status' % id))
```

Comment:
I'm pretty sure both `deployments_update_url` and `deployments_log` should use `device_url`, otherwise you end up using integrations API endpoints. Perhaps add `device_` prefix to both.